### PR TITLE
Reliability analysis fixes

### PR DESCRIPTION
**Pull request by TomiSzucs**
- Visuals in internal consistency reliabiltiy analysis now show reverse items accordingly
- JASP validation added for Cronbach's alpha CI
- Formatting and text fixes

### CODE REVIEW

**Review by krajcsi — CHANGES_REQUESTED** [review]
Thanks for the fixes! Please, check the reversing method

File: cogstat/cogstat.py [hunk 1]
```diff
@@ -968,27 +968,31 @@ def reliability_internal(self, var_names=None, reverse_items=None):
         title = '<cs_h1>' + _('Internal consistency reliability') + '</cs_h1>'
         title += '\n' + _('Reliability of items: ') + ', '.join('%s (%s)' % (var, meas)
                                                                   for var, meas in zip(var_names, meas_levels))
+
+        data = pd.DataFrame(self.data_frame[var_names].dropna())
         if reverse_items:
+            data = data.copy()
```
[the remark this answers is not in the document]
Comment (TomiSzucs) [reply cogstat/cogstat.py]: It was added in a previous version of the code to solve an error I encountered. I tested it now, and indeed it is no longer needed and I deleted it.

File: cogstat/cogstat_stat.py [hunk 2]
```diff
@@ -1257,15 +1257,13 @@ def compare_groups_effect_size(pdf, dependent_var_name, groups, meas_level, samp
 ### Reliability analyses ###
 
 
-def reliability_internal_calc(data, reverse_items=None, sample=True):
+def reliability_internal_calc(data, sample=True):
     """
     Calculate internal consistency reliability using Cronbach's alpha and its confidence interval.
```
[the remark this answers is not in the document]
Comment (TomiSzucs) [reply cogstat/cogstat_stat.py:1262]: Added to the docstring of the function for clarity.

File: cogstat/cogstat_stat.py [hunk 3]
```diff
@@ -1347,9 +1347,10 @@ def reliability_interrater_calc(data, targets=None, raters=None, ratings=None, r
 
     pop_result_df = pd.DataFrame(icc.loc[slice_row][['ICC', 'CI95%']])
     pop_result_df['ICC'] = pop_result_df['ICC'].map('{:,.3f}'.format)
-    # TODO pop_result_df['CI95%'] = pop_result_df['CI95%'].map('{:,.3f}'.format)
+    pop_result_df['CI95%'] = [['%0.3f, %0.3f' % (ci[0], ci[1])] for ci in pop_result_df['CI95%']]
```
[the remark this answers is not in the document]
Comment (TomiSzucs) [reply cogstat/cogstat_stat.py:1351]: Indeed, it does :)

**Comment by krajcsi** [comment]
Great, thanks for the fixes!

**Pull request merged**